docs/source/accelerators/hpu.rst

```diff
@@ -0,0 +1,204 @@
+
+You can execute the ops in FP32 or BF16 precision. The HMP package modifies the python operators to add the appropriate cast operations for the arguments before execution.
+The default settings enable users to easily enable mixed precision training with minimal code.
+
+In addition to the default settings in HMP,  users also have the option of overriding these defaults and providing their own BF16 and FP32 operator lists.
+
+For more details, please refer `<https://docs.habana.ai/en/latest/PyTorch_User_Guide/PyTorch_User_Guide.html#pytorch-mixed-precision-training-on-gaudi>`_.
+
+
+.. _pytorch_lightning_examples:
+
+Getting Started with Lightning on Gaudi
+=======================================
+
+This section describes how to train models using PyTorch Lightning with Habana Gaudi.
+
+More Lightning HPU examples can be found in  pl_examples (`<https://github.com/PyTorchLightning/pytorch-lightning/pl_examples/hpu_examples/ >`)
+
+Enabling Lightning with Single Gaudi HPU
+----------------------------------------
+
+The below snippet shows an example model using MNIST with single Habana Gaudi device:
+
+.. code-block:: python
+
+    import habana_frameworks.torch.core as htcore
+
+
+    class LitClassifier(pl.LightningModule):
+        def __init__(self):
+            super(LitClassifier, self).__init__()
+
+        ...
+
+
+    # Init our model
+    model = LitClassifier()
+
+    # Init DataLoader from MNIST Dataset
+    dm = MNISTDataModule(batch_size=batch_size)
+
+    ...
+
+    num_hpus = 1
+
+    # enable HPU strategy for single device, with mixed precision using default HMP settings
+    hpustrat_1 = SingleHPUStrategy(device=torch.device("hpu"), precision_plugin=HPUPrecisionPlugin(precision=16))
+
+    # Initialize a trainer with 1 HPU accelerator
+    trainer = pl.Trainer(accelerator="hpu", devices=num_hpus, strategy=hpustrat_1)
+
+    # Train the model ⚡
+    trainer.fit(model, datamodule=dm)
+
+
+Enabling Lightning with 8 Gaudi HPUs (distributed)
+--------------------------------------------------
+
+The below snippet shows an example model using MNIST with 8 Habana Gaudi devices:
+
+.. code-block:: python
+
+    import habana_frameworks.torch.core as htcore
+
+
+    class LitClassifier(pl.LightningModule):
+        def __init__(self):
+            super(LitClassifier, self).__init__()
+
+        ...
+
+
+    # Init our model
+    model = LitClassifier()
+
+    # Init DataLoader from MNIST Dataset
+    dm = MNISTDataModule(batch_size=batch_size)
+
+    ...
+
+    num_hpus = 8
+
+    # setup parallel strategy for 8 HPU's
+    hpustrat_8 = HPUParallelStrategy(
+        parallel_devices=[torch.device("hpu")] * num_hpus,
+        precision_plugin=HPUPrecisionPlugin(precision=16),
+    )
+
+    # Initialize a trainer with 1 HPU accelerator
+    trainer = pl.Trainer(accelerator="hpu", devices=num_hpus, strategy=hpustrat_8)
```

Comment:
same here. should this be the same as:
```py
trainer = pl.Trainer(accelerator="hpu", devices=8, precision=16)
```
?

Reply:
To expose hmp_params, there was a request to create backend specific plugin. Hence the change. The test is just to showcase different usage mechanics